components/schema-migrator/main.go

```diff
@@ -96,8 +129,58 @@ func main() {
 	}
 
 	if err != nil {
-		fmt.Printf("Error during migration, %s\n", err)
+		return fmt.Errorf("during migration: %s", err)
+	}
+
+	return nil
+}
+
+func quitCloudSqlProxy() error {
+	fmt.Println("# QUITTING CLOUD SQL PROXY #")
+	matches, err := filepath.Glob("/proc/*/exe")
+	if err != nil {
+		return fmt.Errorf("while reading cloud_sql_proxy: %s", err)
```

Comment:
reading process list - at this stage. Looking for cloud_sql_proxy process is below.